ci/jobs/satellite6-pulp.yaml

```diff
@@ -0,0 +1,187 @@
+# This job automates the process of deploying Satellite 6 on OpenStack instances
+# and upgrading the Pulp for Robottelo and Pulp Smash tests.
+# Prerequisites:
+#     1) Activate account on OpenStack platform.
+#     2) Download the OpenStack RC file from Access&Security>API Access.
+#     3) Create a secret credential file <os-secret.sh> with the following format:
+#         export OS_AUTH_URL=OS_AUTH_URL
+#         export OS_USERNAME=OS_USERNAME
+#         export OS_PASSWORD=password
+#         export OS_TENANT_NAME='"US.ENG Perf R&D"'
+#     4) Upload this secret file to Jenkins UI, for a credentials-binding id.
+#     5) Create a secret credential file <jenkins-secret.sh>:
+#         export REMOTE_JENKINS_USERNAME=username
+#         export REMOTE_JENKINS_API_TOKEN=token
+#         export REMOTE_JENKINS_URL=<remote-jenkins-server-host>
+#     4) Upload this secret file to Jenkins UI, for a credentials-binding id.
+# The automation steps are as following:
+#     1) prebuild to create OS1 instances of RHEL 7.2;
```

Comment:
The steps here can be more generic since the same job may be used for other OS versions for example

1. Create an instance on Open Stack
2. Run satellite6-installer job to install latest Satellite 6 downstream build
3. Run satellite6-pulp-upgrade job to upgrade the version of Pulp
4. Run satellite6-standalone-automation